packages/web/components/twitter-section/twitter-section.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+function TwitterSection() {
+  const tweetContent = `Get ready for Osmocon 2023!\n\nJoin us in Paris for a jam-packed day, connect with some of the greatest industry innovators, and explore the future of Cosmos.\n\nDon't miss out!\n\nGet your tickets now:`;
+
+  return (
+    <div className="flex flex-1 flex-col items-start gap-6 self-stretch rounded-5xl border border-osmoverse-800 bg-osmoverse-900 py-10 px-8">
```

Comment:
Can we reduce the padding like we do for "Related assets" and "Token details" sections?

Reply:
Ofc for mobile